ibm/mas_devops/roles/suite_app_config/README.md

```diff
@@ -180,6 +180,11 @@ Optional. Flag indicating if manage demodata should be loaded or not.
 - Environment Variable: `MAS_APP_SETTINGS_DEMODATA`
 - Default: `false` (do not load demodata)
 
+### "{{ mas_app_settings_db2vargraphic | bool }}"
+Optional. Flag indicating if VARGRAPHIC (if true) or VARCHAR (if false) is used.
```

Comment:
Can you add details about what the flag does ... what's the difference is vargraphic is true or false?  Is there a page in the product doc we can link to?

Reply:
It sets the flag in ManageWorkspace which controls how DB2 was deployed - with VARCHAR or VARGRAPHIC column. For new deployments it will be always be VARGRAPHIC but for upgrades (especially old ones 7.1->7.6->MAS) it may be still VARCHAR.